uint32 Home = 0
uint32 PanLeft = 1
uint32 PanRight = 2
uint32 TiltUp = 3
uint32 TiltDown = 4
uint32 PanScan = 5
uint32 TiltScan = 6

uint32 command
